### PR TITLE
ADO 168576 - Appropriate markup to fix accessibility issues on questions and results pages

### PR DESCRIPTION
**Pull request by alex-solo**
### Description

- Marital status options should be a list (was using p tags)
- bold text should be in a strong element for readers to know it's impportant

### CODE REVIEW

File: i18n/web/en.ts [hunk 1]
```diff
@@ -308,7 +308,7 @@ const en: WebTranslations = {
   resultsPage: {
     header: 'Table of estimated monthly amounts',
     general:
-      'The following is only an estimate of your eligibility and monthly payments <span style="font-weight: bold;">based on current rates</span>. Amounts may increase with the cost of living. Changes in your circumstances may also impact your results.',
+      'The following is only an estimate of your eligibility and monthly payments <strong style="font-weight: bold;">based on current rates</strong>. Amounts may increase with the cost of living. Changes in your circumstances may also impact your results.',
```
Comment (Maxim-Lam) [inline i18n/web/en.ts]: Doesn't <strong> tag already make font-weight: bold? might be a redundancy?

File: i18n/web/fr.ts [hunk 2]
```diff
@@ -312,7 +312,7 @@ const fr: WebTranslations = {
   },
   resultsPage: {
     header: "Tableau des résultats d'estimation",
-    general: `Les résultats suivants ne sont qu'une estimation de votre admissibilité et de vos paiements mensuels <span style="font-weight: bold;">basée sur les montants actuels</span>. Ceux-ci peuvent augmenter avec le coût de la vie. Des changements dans votre situation pourraient aussi modifier vos résultats`,
+    general: `Les résultats suivants ne sont qu'une estimation de votre admissibilité et de vos paiements mensuels <strong style="font-weight: bold;">basée sur les montants actuels</strong>. Ceux-ci peuvent augmenter avec le coût de la vie. Des changements dans votre situation pourraient aussi modifier vos résultats`,
```
Comment (Maxim-Lam) [inline i18n/web/fr.ts]: Same comment as above

**Review by Maxim-Lam — CHANGES_REQUESTED** [review]
Looks good. Just the one comment to consider.

File: i18n/web/en.ts (hunk 1 above)
Comment (Maxim-Lam) [reply i18n/web/en.ts]: sorry, wrote "<" strong ">" tag*

File: i18n/web/fr.ts (hunk 2 above)
Comment (alex-solo) [reply i18n/web/fr.ts]: good catch, got rid of styles

**Review by Maxim-Lam — APPROVED** [review]
Looks good!

**Pull request merged**